common/src/main/java/io/seata/common/ConfigurationKeys.java

```diff
@@ -946,6 +946,11 @@ public interface ConfigurationKeys {
      */
     String CLIENT_METADATA_MAX_AGE_MS = CLIENT_PREFIX + "metadataMaxAgeMs";
 
+    /**
+     * The constant CLIENT_METADATA_MAX_AGE_MS.
+     */
+    String TOKEN_VALID_TIME_MS_KEY = "tokenValidityInMilliseconds";
```

Comment:
Take a look at the RegistryServiceImpl in Nacos. These configuration items are written directly into the specific implementation class, rather than being placed in configuration keys. Additionally, these configurations need to be added to the corresponding properties in the client module of the spring-boot-autoconfig.
参考下nacos那边的registryserviceimpl,这些配置项都是写到具体的实现类中的,而不是放到configurationkeys中,然后你这个配置还得加到对应的spring-boot-autoconfig的clientmodule里的对应properties中

Reply:
fixed